tests/unit/modules/test_user_info.py

```diff
@@ -23,7 +23,7 @@ def test_get_all_users(self, mocker):
             user_info.main()
 
         _client, path = get_mock.call_args[0]
-        assert path == "/users"
+        assert path == "/api/core/v2/users"
         assert context.value.args[0]["objects"] == [1, 2, 3]
 
     def test_get_single_mutator(self, mocker):
```

Comment:
Ugh, I just noticed this.
I know this is totally unrelated to the changes you made in this PR, but could you please just rename it to `test_get_single_user`?